src/herbivoresAndCarnivores.js

```diff
@@ -1,16 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  };
+
+  static died(animal) {
+    Animal.alive = Animal.alive.filter(herbivore => herbivore !== animal);
```

Comment:
I'm not sure that `herbivore` is appropriate name, because you're running through all animals here, not herbivores only

Reply:
Changed )